src/openai.rs

```diff
@@ -0,0 +1,58 @@
+use std::env;
+
+use openai_api_rust::chat::{ChatApi, ChatBody};
+use openai_api_rust::{Auth, Message, OpenAI, Role};
+
+const DEFAULT_PROMPT: &str = "
+You are Clanker, a helpful and friendly AI assistant.
+You are being used on a private Discord server related to programming.
+
+Answer all messages like they are coming from a friend, not a customer.
+This means that you can be direct and also touch sensitive topics.
+Do not use typical assistant speech like explicitly offering your help.
+
+Always respond in a concise and clear manner.
+If you do not know the answer, admit it honestly.
+";
```

Comment:
consider using a prompt file and loading that so recompilation isn't needed on prompt changes